README.md

```diff
@@ -7,20 +7,21 @@ the cursor and you will see the PDF file updated). Currently,
 vim-latex-live-preview only support UNIX-like systems. [Please let me know if
 you have any suggestions.](.github/CONTRIBUTING.md)
 
-Table of Contents
------------------
+![Screenshot with Evince](misc/screenshot-evince.gif)
+
+## Table of Contents
 
 - [Installation](#installation)
 - [Usage](#usage)
 - [Configuration](#configuration)
-- [Known issues](#known-issues)
-- [Screenshot](#screenshot)
+- [Known Issues](#known-issues)
+- [Contributing](#contributing)
+- [Release History](#release-history)
 
-Installation
-------------
+# Installation
```

Comment:
It should be `##` instead of single `#`.
Actually, this is why I personally prefer using `---` 😆